src/components/DeploymentScheduleToggle.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <p-tooltip text="Pause or resume this schedule">
+    <p-toggle v-if="deployment.can.update" v-model="internalValue" :disabled="loading" :loading="loading" />
```

Comment:
I don't believe there is a `loading` prop here. Probably fine to simply disable it while loading.